subprojects/docs/src/docs/userguide/console.adoc

```diff
@@ -0,0 +1,147 @@
+----
+
+[[sec:console_work_in_progress_display]]
+==== Work in-progress display
+
+The console provides a more fined-grained view of the actual work being performed. This information can be found directly underneath the build progress bar. Each line represents a worker thread or process that can perform work in parallel e.g. resolving dependencies, executing a task action. If an available worker is not utilized then it is marked with `IDLE`. The number of available workers equates to the number of processors on the machine executing the build. If needed a user can provide the number of workers through the <<gradle_command_line,command line>>  option `--max-workers`.
+
+The following portion of the console output shows the work in-progress display in action:
+
+----
+<==========---> 77% EXECUTING [10s]
+> :codeQuality:classpathManifest > Resolve dependencies :codeQuality:runtimeClasspath
+> :ivy:classpathManifest > Resolve dependencies :ivy:runtimeClasspath
+> IDLE
+> :antlr:classpathManifest > Resolve dependencies :antlr:runtimeClasspath
+> :scala:compileJava > Resolve dependencies :scala:compileClasspath
+> :buildInit:classpathManifest > Resolve dependencies :buildInit:runtimeClasspath
+> :jacoco:classpathManifest > Resolve dependencies :jacoco:runtimeClasspath
+> IDLE
+----
+
+[[sec:console_build_result]]
+==== Build result
+
+After finishing the build, the console will render the outcome of the build and a ratio of work performed vs. work avoided. The outcome indicates if the build was successful or failed. It also displays the elapsed time it took to run the overall build. The work ratio gives the user an idea about the number of performed tasks or tasks that did not need to be executed. The intention of the summarized view is to avoid too much information in the console output during the build.
+
+----
+BUILD SUCCESSFUL in 2m 10s
+411 actionable tasks: 381 executed, 30 up-to-date
+----
+
+[[sec:console_build_output]]
+==== Build output
+
+Build output is rendered right above the build progress bar. Typical output includes log messages, messages from the standard output and error streams, compile warnings, test output events and many more.
+
+Starting with version 4.0 of Gradle, the console output has been reduced dramatically. The console does not render every task with its build status anymore - it only renders output that is emitted during a build lifecycle phase or a task execution. Avoiding an abundance of information keeps output as readable as possible. Gradle grouped output originating from a specific context e.g. all warnings from a compilation task. Grouped output is especially useful in a project setup that enables parallel project execution as it prevents concurrent interleaved messages to be interleaved without clearly indicating its origin (see xref:parallel_execution[]).
+
+NOTE: To render console output with the formatting of pre-4.0 versions of Gradle, add the <<gradle_command_line,command line>> option `--console=plain`.
```

Comment:
I would turn this around.  

NOTE: Grouped console output and reduced console output only occurs with interactive and _rich console_ command-lines.  Continuous integration servers and builds using `--console=plain` will see console output similar to pre-Gradle 4.0.